install/terraform/modules/gke/variables.tf

```diff
@@ -33,6 +33,7 @@ variable "cluster" {
     "network"           = "default"
     "subnetwork"        = ""
     "kubernetesVersion" = "1.18"
+    "windows" = "false"
```

Comment:
nit: can you fix the spacing so that the = signs line up?

Reply:
Looks like it needs a quick: https://www.terraform.io/docs/cli/commands/fmt.html 😄